Docs/sphinx_documentation/source/Visualization.rst

```diff
@@ -7,68 +7,91 @@ Amrvis
-   compiler suite you have.
+   To build, ``cd`` into ``Amrvis/``, and edit ``GNUmakefile`` by setting the variable
+   ``COMP`` to your compiler suite.
 
-   Type ``make DIM=1``, ``make DIM=2``, or ``make DIM=3`` to build, resulting
-   in an executable that looks like amrvis2d...ex.
+   Type ``make DIM=1``, ``make DIM=2``, or ``make DIM=3`` to build. The result is
+   an executable that looks like ``amrvis2d.<ver>.ex``.
 
-   If you want to build amrvis with ``DIM=3``, you must first download and build
-   ``volpack``:
+   |
 
-   .. highlight:: console
+   *3D Data Visualization with Volpack*
 
-   ::
+   If you want to build Amrvis with ``DIM=3`` for display of 3-dimensional data,
+   you must first download and build ``volpack``. This can be done by cloning
+   the repository:
+
+   .. code-block:: console
 
        git clone https://ccse.lbl.gov/pub/Downloads/volpack.git
 
-   Then cd into volpack/ and type ``make``.
+   After downloading, ``cd`` into ``volpack/`` and type ``make``.
 
-   Note: Amrvis requires the OSF/Motif libraries and headers. If you don't have
+   |
+
+   **Note**:sup:`1`: Amrvis requires the OSF/Motif libraries and headers. If you don't have
    these you will need to install the development version of motif through your
    package manager.  ``lesstif`` gives some functionality and will allow you to
-   build the amrvis executable, but Amrvis may exhibit subtle anomalies.
+   build the Amrvis executable, but Amrvis may exhibit subtle anomalies.
 
    On most Linux distributions, the motif library is provided by the
-   ``openmotif`` package, and its header files (like Xm.h) are provided by
+   ``openmotif`` package, and its header files (like ``Xm.h``) are provided by
    ``openmotif-devel``. If those packages are not installed, then use the
    OS-specific package management tool to install them.
 
-   You may then want to create an alias to amrvis2d, for example
+   |
 
-   .. highlight:: console
+   **Note**:sup:`2`: These instructions assume that the install directories
+   for Amrvis and volpack share the same parent directory. To install volpack
+   in a different location specify the location of volpack in Amrvis's
+   ``GNUmakefile`` by changing the variable ``VOLPACKDIR`` to the desired location.
 
-   ::
+   |
+
+   After building you may want to create an alias for convenience.
+   To do this type,
 
-       alias amrvis2d /tmp/Amrvis/amrvis2d...ex
+   .. code-block:: console
 
-#. Run the command ``cp Amrvis/amrvis.defaults ~/.amrvis.defaults``.  Then, in
-   your copy, edit the line containing "palette" line to point to, e.g.,
+       alias amrvis2d /tmp/Amrvis/amrvis2d.<ver>.ex
+
+   |
+
+#. **Configure**:
+
+   Amrvis will look for the configuration file ``.amrvis.defaults`` in your
+   home folder. Run the command ``cp Amrvis/amrvis.defaults ~/.amrvis.defaults`` to
+   copy it there.  Then, in
+   your copy, edit the line containing "palette" to point to the color palette
```

Comment:
Should we explicitly state here that there are initial .amrvis.defaults and palette files in the directory? Seems to have been a little confusion about that recently.

Reply:
I rewrote this paragraph to make those things more explicit. Please take a look at the new version.